client/wfnews-war/src/main/angular/src/app/components/panel-wildfire-stage-of-control/panel-wildfire-stage-of-control.component.ts

```diff
@@ -79,4 +93,30 @@ export class PanelWildfireStageOfControlComponent extends CollectionComponent im
           }
     }
 
+    calculateDistance(lat: string, long: string){
```

Comment:
Best to pass in Numbers, rather than casting internally. A calculate function would expect numeric latitudes and longitudes